mmv1/products/compute/GlobalForwardingRule.yaml

```diff
@@ -484,6 +484,37 @@ properties:
     update_verb: :POST
     update_url: 'projects/{{project}}/global/forwardingRules/{{name}}/setTarget'
     diff_suppress_func: 'tpgresource.CompareSelfLinkRelativePaths'
+  - !ruby/object:Api::Type::Array
+    name: 'serviceDirectoryRegistrations'
+    description: |
+      Service Directory resources to register this forwarding rule with.
+
+      Currently, only supports a single Service Directory resource.
+    min_size: 0
+    max_size: 1
+    immutable: true
+    default_from_api: true
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'namespace'
+          description: |
+            Service Directory namespace to register the forwarding rule under.
+          immutable: true
+          default_from_api: true
+        - !ruby/object:Api::Type::String
+          name: 'service'
```

Comment:
Can you add a test that specifies this field?

Reply:
It's currently not applicable for global_forwarding_rule although globalForwardingRule API provides it https://cloud.google.com/compute/docs/reference/rest/v1/globalForwardingRules/insert#request-body. Meaning no globalForwardingRule will have this service field set. But in the future, it might be in use.
What is the norm here? Should I remove it or keep it in sync with the API?

Reply:
I would remove it for now, we can add it once support is added in the API

Reply:
Removed